### ISSUE TITLE
Unable to Access anything in superset request throwing 401

### ISSUE THREAD

**Issue by AnkitSingh02** [apache/incubator-superset#9016]
I have installed superset on my mac using python venv, have also successfully configured a secure login of my company using OAUTH configuration.

After logging in into the dashboard(using oauth), I am not able to perform/choose any options: 

PFB the logs from debugger mode : 

2020-01-25 00:19:52,808:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:19:52] "GET / HTTP/1.1" 302 -
2020-01-25 00:19:52,814:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:19:52] "GET /superset/welcome HTTP/1.1" 302 -
2020-01-25 00:19:52,931:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:19:52] "GET /login/ HTTP/1.1" 200 -
**2020-01-25 00:19:53,543:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:19:53] "GET /superset/csrf_token/ HTTP/1.1" 401 -**

**2020-01-25 00:20:01,134:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:01] "GET /login/authn?next= HTTP/1.1" 302 -**
2020-01-25 00:20:01,285:DEBUG:flask_oauthlib:Prepare oauth2 remote args {'code': '<Code_For AuthN>', 'client_secret': '<SECRET>', 'redirect_uri': '<REDIRECT URL>'}
2020-01-25 00:20:01,285:DEBUG:flask_oauthlib:Request '<AUTH TOKEN URL>' with 'POST' method
2020-01-25 00:20:01,449:DEBUG:flask_oauthlib:Request '<BASE URL>' with 'GET' method
{'username': '<USer Name>', 'email': '<email Id>'}
2020-01-25 00:20:01,597:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:01] "GET /oauth-authorized/authn?code=<Complete AUthN code> HTTP/1.1" 302 -
2020-01-25 00:20:01,602:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:01] "GET / HTTP/1.1" 302 -
2020-01-25 00:20:01,723:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:01] "GET /superset/welcome HTTP/1.1" 200 -

**2020-01-25 00:20:01,952:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:01] "GET /superset/recent_activity/3/?limit=50 HTTP/1.1" 401 -
2020-01-25 00:20:01,952:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:01] "GET /superset/fave_dashboards/3/ HTTP/1.1" 401 -
2020-01-25 00:20:01,953:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:01] "GET /dashboardasync/api/read?_oc_DashboardModelViewAsync=changed_on&_od_DashboardModelViewAsync=desc HTTP/1.1" 401 -
2020-01-25 00:20:01,953:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:01] "GET /superset/fave_slices/3/ HTTP/1.1" 401 -**
2020-01-25 00:20:07,204:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:07] "GET /users/userinfo/ HTTP/1.1" 302 -
2020-01-25 00:20:07,212:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:07] "GET /login/?next=http%3A%2F%2Flocalhost%3A8088%2Fusers%2Fuserinfo%2F HTTP/1.1" 302 -
2020-01-25 00:20:07,216:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:07] "GET / HTTP/1.1" 302 -
2020-01-25 00:20:07,245:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:07] "GET /superset/welcome HTTP/1.1" 200 -
**2020-01-25 00:20:07,435:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:07] "GET /superset/fave_dashboards/3/ HTTP/1.1" 401 -
2020-01-25 00:20:07,439:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:07] "GET /superset/recent_activity/3/?limit=50 HTTP/1.1" 401 -
2020-01-25 00:20:07,439:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:07] "GET /superset/fave_slices/3/ HTTP/1.1" 401 -
2020-01-25 00:20:07,446:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:07] "GET /dashboardasync/api/read?_oc_DashboardModelViewAsync=changed_on&_od_DashboardModelViewAsync=desc HTTP/1.1" 401 -**
2020-01-25 00:20:09,698:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:09] "GET /superset/sqllab HTTP/1.1" 302 -
2020-01-25 00:20:09,707:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:09] "GET /login/?next=http%3A%2F%2Flocalhost%3A8088%2Fsuperset%2Fsqllab HTTP/1.1" 302 -
2020-01-25 00:20:09,711:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:09] "GET / HTTP/1.1" 302 -
2020-01-25 00:20:09,745:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:09] "GET /superset/welcome HTTP/1.1" 200 -
**2020-01-25 00:20:09,944:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:09] "GET /superset/fave_dashboards/3/ HTTP/1.1" 401 -
2020-01-25 00:20:09,948:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:09] "GET /dashboardasync/api/read?_oc_DashboardModelViewAsync=changed_on&_od_DashboardModelViewAsync=desc HTTP/1.1" 401 -
2020-01-25 00:20:09,949:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:09] "GET /superset/fave_slices/3/ HTTP/1.1" 401 -
2020-01-25 00:20:09,949:INFO:werkzeug:127.0.0.1 - - [25/Jan/2020 00:20:09] "GET /superset/recent_activity/3/?limit=50 HTTP/1.1" 401 -**

Note : 
-- After Clicking on Profile or choosing options its throwing 401.
-- The LogIn Authorization is done by ouath integration.
-- When I wasnt using oauth security for authentication was able to login using normal admin credentials and update data source and add dashboards.

The Screen is simply Blank : 

<img width="1667" alt="Screenshot 2020-01-25 at 12 25 56 AM" src="https://user-images.githubusercontent.com/16684067/73096010-da6c2f00-3f09-11ea-8d9b-5c331dbe9630.png">

Thanks in advance

**Comment by songlipeng2003** [comment]
same error

**Comment by songlipeng2003** [comment]
I find reason,  docker-ini.sh fail.
superset load_examples fail in step 3, and superset init fail in step 4.

**Comment by moyee** [comment]
> I find reason, docker-ini.sh fail.
> superset load_examples fail in step 3, and superset init fail in step 4.

How to solve it? reinstall ?

**Comment by willbarrett** [comment]
Hi @songlipeng2003 and @moyee. My recommendation would be to restart your docker-compose setup to re-start the initialization process. If that doesn't work, try `docker-compose down -rmi -v` to remove all of the existing setup followed by another `docker-compose up`. Additionally, please ensure that Docker has 6 to 8 GB of RAM available if you're using Docker for Mac or a virtual machine. If you are able to post logs from the Docker process, or the error messages you run into during that process, that will help us debug the issue. Unfortunately web server logs are not useful in this case, as the issue is in the startup process. I hope that helps!

**Comment by songlipeng2003** [comment]
@moyee I update cli.py disable utils.get_example_database() .

**Comment by ssYkse** [comment]
Hi, I am facing the same issue in a different setup. 
At work we have an OpenShift 4.2 Cluster, and I have installed via Helms stable/superset. I had to change the running user/fsGroup however got everythign to run. 

After going to the webpage for the first time I was greated by https://github.com/apache/incubator-superset/issues/7143 this error. Running superset db upgrade fixed it.

Afterwards the 'admin admin' user didn't work. So I created a new one using fabmanager create-admin --app superset, after which I could login.

However, after the login page I am exactly at this Issue.
